docs/pangeo-showcase.rst

```diff
@@ -28,32 +28,32 @@ Spring 2023 Showcase
      - Title
    * - 2023-02-01 4PM EST
      - Tom Nicholas, Columbia University 
-     - Xarray-Datatree: Hierarchical Data Structures for Multi-Model Science
+     - `Xarray-Datatree: Hierarchical Data Structures for Multi-Model Science <https://discourse.pangeo.io/t/wednesday-february-1st-xarray-datatree-hierarchical-data-structures-for-multi-model-science/3207>`_ |Nicholas DOI Badge|
    * - 2023-02-08 12PM EST
-     - Alex Kerney, Gulf of Maine Research Institute 
+     - `Alex Kerney, Gulf of Maine Research Institute <https://youtu.be/XfzOxpHRDnw>`_
      - Mental Health for Geoscientists
    * - 2023-02-15 4PM EST
      - Tim Crone, Columbia University
-     - Lessons learned teaching Pangeo in the classroom
+     - `Lessons learned teaching Pangeo in the classroom <https://discourse.pangeo.io/t/wednesday-february-15th-lessons-learned-teaching-pangeo-in-the-classroom/3208>`_ |Crone DOI Badge|
    * - 2023-02-22 12PM EST
-     - Ramon Ramirez-Linan, Navteca
+     - `Ramon Ramirez-Linan, Navteca <https://youtu.be/AknQEtyoc6U>`_
```

Comment:
Is there a reason why we have the direct YouTube link for the speaker name here? In other cases we seem to have a link to the Discourse post, which includes the link to the YouTube video.

Reply:
I was still collecting the final copies of slides for the archive post for those ones, but they were on top of it this morning so those have been appropriately updated.

Reply:
Wonderful! Thanks!